.github/workflows/mavenCi.yml

```diff
@@ -0,0 +1,59 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  workflow_dispatch:
+  push:
+  pull_request:
+
+
+
+jobs:
+  job1:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'corretto'
+          cache: maven
+      - name: Build
+        run: |
+          mvn package -B clean verify --file pom.xml
```

Comment:
I guess we do not need to specify the pom file - as its the default one?

Reply:
--file pom.xml is not necessary and is removed